cmd/slurmCheck/slurmCheck.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	//"io"
+	"log"
+	"strings"
+	//"time"
+)
+
+type slurmCheckArray struct {
+	begin      string // whatever is in front of the squiggles
+	outToCheck string // this is the file to be checked
+	checkType  string // this is the code for how we check the output files
```

Comment:
What I understood was that the inputFancyFile.txt already has info on whether each slurm job output file meets the checks (empty, line, etc.), and this program parses inputFancyFile.txt so you can see that info more clearly. Is this right? Or is this program doing the checks (empty, line, etc.)?

Reply:
This program is going to submit the job array to SLURM, and do the check. Since there are check options and depending on the software being used, the output file could be argument 2, 3, etc, the input file needs to be different than a normal job array. We use squiggles "{ }" in this specialized job array file to tell slurmCheck where to look, this is the file I'm calling the "fancyFile.txt". The user must tell slurmCheck which file is the expected output file and which type of "check" the user wants to be performed.

Reply:
Thanks! Are you still working on the part where the program interacts with SLURM to submit the job array and do the check? Or did you already write it in this PR and I missed it (I didn't find sbatch or anything like that in the uncommented code)?

Reply:
I wanted to submit my PRs in increments to introduce the idea and gett feedback as I'm going versus at the end. This is only parsing the file, which will likely also update as I write the code for the other features.

Reply:
OK got it! Thanks for the explanation! Excited to see how gonomics will now be programmed to interact with SLURM